src/sentry/static/sentry/app/views/organizationIntegrations/integrationStatus.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import styled from '@emotion/styled';
+import CircleIndicator from 'app/components/circleIndicator';
+import theme from 'app/utils/theme';
+import {t} from 'app/locale';
+import space from 'app/styles/space';
+import {withTheme} from 'emotion-theming';
+
+import {colors} from './constants';
+
+type StatusProps = {
+  status: string;
```

Comment:
@NisanthanNanthakumar We definitely should type this based off the constants you added in `organizationIntegrations/constants.tsx`